### DISCUSSION TITLE
How to extend a notification full screen, stuck to the bottom?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#12656] [General - Components / Directives / etc] [answered]
I use the [`Notification`](https://quasar.dev/quasar-plugins/notify) plugin and it works great.

I would like to customize it to change its visual aspect:
- extend it to the full width of the screen (or a relative width independent on the content)
- stick it to the bottom of the screen (so that it appears as it does now, but without a margin at the bottom, ideally with square corners to match the browser edges)

Is this something that is configurable within Quasar, or should I look for another notification toaster to make it simpler?

**Comment by MilosPaunovic** [comment, marked as answer]
I would suggest that you stick with Quasar notification plugin and not install another dependency, as it will only bloat your codebase. The CSS styling system used is extremly simple to modify, so, to achive what you've described only takes two classes.

[https://codepen.io/milospaunovic/pen/podGKRj](https://codepen.io/milospaunovic/pen/podGKRj
)

**Reply by wsw70** [reply]
Thank you. I did not consider overwriting the CSS classes built into Quasar. In retrospect, this is the straightforward approach. Thanks for this very complete answer (and the code)

**Reply by rstoenescu** [reply]
Also check the many class/style props. You could set some defaults on the Notify plugin then focus on the CSS of your custom classes, rather than overwriting Notify directly.
